conf/messages.en

```diff
@@ -2454,21 +2454,21 @@ memberPensionPayments.memberList.pagination.label = Showing {0} - {1} of {2} mem
 memberPensionPayments.memberList.pensionPaymentsReported = Pension payments reported
 memberPensionPayments.memberList.noPensionPayments = No pension payments
 
-totalAmountPensionPayments.title = What was the total amount of pension payments received by the member during the period of this return, in pounds and pence?
-totalAmountPensionPayments.heading = What was the total amount of pension payments received by {0} during the period of this return, in pounds and pence?
-totalAmountPensionPayments.error.required = Enter the total amount of pension payments, in pounds and pence
-totalAmountPensionPayments.error.invalid = Total amount of pension payments must be an amount of money, like 123 or 123.45
-totalAmountPensionPayments.error.tooSmall = Total amount of pension payments must be 0.01 or more
-totalAmountPensionPayments.error.tooLarge = Total amount of pension payments must be 999999999.99 or less
+totalAmountPensionPayments.title = What was the taxable amount of pension payments received by the member during the period of this return, in pounds and pence?
```

Comment:
@Vamshi-Muniganti @BabuRupoll7623 @harishmuddegowda Do we need to update acceptance tests for this title change?